### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by chevdor**
This PR is a backport of a commit from the 0.9.10 release allowing to fix the blocking spelling issues.

### CODE REVIEW

**Review by tomusdrw — APPROVED** [review]
This should be done by a `git subtree` update to avoid conflicts in the future, but let's unblock whatever is blocking you :)

**Review by svyatonik — APPROVED** [review]

**Comment by chevdor** [comment]
Thank you @tomusdrw. Please note however that this is NOT blocking me as I have the fix in the release branch anyway. I just wanted to backport it here to ensure it is kept in sync.
So if you see complications merging this PR, we can also drop it.

**Comment by tomusdrw** [comment]
> So if you see complications merging this PR, we can also drop it.

It's alright, should be resolved automatically anyway. I plan to update the bridges code this/next week anyway.

File: bridges/.config/lingua.dic [hunk 1]
```diff
@@ -65,6 +65,8 @@ isn
 io
 js
 keccak256/M
+keypair/MS
+KeyPair
```
Comment (drahnr) [inline bridges/.config/lingua.dic:69]: This should be code right? So we should not whitelist it, but require it to be `\`KeyPair\`` (working on a shortcut for `cargo-spellcheck` to have this in interactive mode).

File: bridges/.config/lingua.dic [hunk 2]
```diff
@@ -101,6 +103,7 @@ parablock/MS
 parachain/MS
 parameterize/D
 pallet_message_lane
+plancks
```
Comment (drahnr) [inline bridges/.config/lingua.dic:106]: :+1:

File: scripts/gitlab/lingua.dic [hunk 3]
```diff
@@ -117,6 +117,7 @@ jaeger/MS
 js
 keccak256/M
 keypair/MS
+KeyPair
```
Comment (drahnr) [inline scripts/gitlab/lingua.dic:120]: CC

**Review by drahnr — APPROVED** [review]
LGTM, we should probably remove `\`KeyPair\`` from the allowed list.

I am also slightly puzzled why bridges has their own spellcheck config :monocle_face:

**Comment by chevdor** [comment]
@drahnr I did not dig into the meaning of those `/MS` but `keypair/MS` appears to accept only `Keypair` and not `KeyPair`. The detected mispelling comes from a comment refering to the type `KeyPair`. If you know how we can modify the dictionnary without the duplication, we can do it now but I think those disctionnary need a major cleanup anyway

File: bridges/.config/lingua.dic (hunk 1 above)
Comment (chevdor) [reply bridges/.config/lingua.dic:69]: No this is a **comment** refering to the type called `KeyPair`.

**Comment by drahnr** [comment]
bot merge

**Pull request merged**